src/proto/Ethereum.proto

```diff
@@ -181,4 +176,7 @@ message SigningOutput {
 
     // error code description
     string error_message = 7;
+
+    // Encoded transaction bytes.
+    bytes pre_hash = 8;
```

Comment:
what does this do? will it be filled for every eth tx?

Reply:
Yes, it is the hash that is normally signed by the private key. We need to expose it because with passkeys signing happens outside of wallet core.